docs/source/model/signatures.rst

```diff
@@ -748,7 +748,51 @@ model signatures in log_model calls when signatures aren't specified.
     Secondly, and just as importantly, it **validates the model's requirements**. This input example is utilized to execute a prediction 
     using the model that is about to be logged, thereby enhancing the accuracy in identifying model requirement dependencies.
     It is **highly recommended** to always include an input example along with your models when you log them.
+    Since MLflow 2.16.0, if the provided input example fails on validation, a warning is issued with an instruction to validate the input example
+    prior to model deployment.
 
+Since MLflow 2.16.0, when logging a model with an input example, there are two files saved into the model's artifacts directory:
+
+- ``input_example.json``: The input example in JSON format.
+- ``serving_input_payload.json``: The input example converted to JSON format for querying a deployed model endpoint.
+
+The following example demonstrates the difference between the two files:
+
+.. code-block:: python
+
+    import mlflow
+
+
+    class MyModel(mlflow.pyfunc.PythonModel):
+        def predict(self, context, model_input, params=None):
+            return model_input
+
+
+    with mlflow.start_run():
+        model_info = mlflow.pyfunc.log_model(
+            python_model=MyModel(),
+            artifact_path="model",
+            input_example={"question": "What is MLflow?"},
+        )
+
+Example files:
+
+ +---------------------------------------+-----------------------------------------+
+ |  input_example.json                   |  serving_input_payload.json             |
+ +---------------------------------------+-----------------------------------------+
+ | .. code-block:: json                  | .. code-block:: json                    |
+ |                                       |                                         |
+ |    {                                  |   {                                     |
+ |        "question": "What is MLflow?"  |       "inputs": {                       |
+ |    }                                  |         "question": "What is MLflow?"   |
+ |                                       |       }                                 |
+ |                                       |   }                                     |
+ |                                       |                                         |
+ +---------------------------------------+-----------------------------------------+
+
+
+``input_example.json`` contains the input example in its original format, while ``serving_input_payload.json`` is a JSON-serialized version of the input example with
+a defined ``key`` (one of ``dataframe_split, instances, inputs or dataframe_records``) that mlflow scoring server requires when `querying a deployed model endpoint <../../cli.html#mlflow-models-serve>`_.
```

Comment:
```suggestion
one of predefined keys (``"dataframe_split"``, ``"instances"``, ``"inputs"``, and ``"dataframe_records"``) that mlflow scoring server requires when `querying a deployed model endpoint <../../cli.html#mlflow-models-serve>`_.
```
super-nit

Reply:
Btw the link for `cli.html` is broken. Can we link to https://mlflow.org/docs/latest/deployment/deploy-model-locally.html#local-inference-server-spec instead, which has more comprehensive guidance?